src/screens/Asl/components/AslTable/index.js

```diff
@@ -56,6 +57,83 @@ const AslTable = props => {
     const [showVideo, setShowVideo] = useState(false);
     const [videoUrl, setVideoUrl] = useState('');
 
+    const wrapStyle = {
+      left: 100,
+      top: 100,
+      width: 500,
+      height: 500
+    }
+    const [style, setStyle] = useState({
+        left: 100,
+        top: 100,
+        width: 400,
+        height: 200
+    })
+    // 初始数据， 因为不需要重新render 所以用 useRef
+    const oriPos = useRef({
+        top: 0, // 元素的坐标
+        left: 0,
+        cX: 0, // 鼠标的坐标
+        cY: 0
+    })
+    const isDown = useRef(false);
+    const direction = useRef();
+
+    function transform(direction, oriPos, e) {
+      const style = {...oriPos.current}
+      const offsetX = e.clientX - oriPos.current.cX;
+      const offsetY = e.clientY - oriPos.current.cY;
+      switch (direction.current) {
+          // 拖拽移动
+          case 'move' :
+              // 元素当前位置 + 偏移量
+              const top = oriPos.current.top + offsetY;
+              const left = oriPos.current.left + offsetX;
+              // 限制必须在这个范围内移动 画板的高度-元素的高度
+              style.top = top
+              style.left = left;
+              break
+          case 'se':
+              style.height += offsetY;
+              style.width += offsetX;
+              break
+      }
+      return style
+    }
+    // 鼠标被按下
+    const onMouseDown = useCallback((dir, e) => {
+        // 阻止事件冒泡
+        e.stopPropagation();
+        // 保存方向。
+        direction.current = dir;
+        isDown.current = true;
+        // 然后鼠标坐标是
+        const cY = e.clientY; // clientX 相对于可视化区域
+        const cX = e.clientX;
+        oriPos.current = {
+            ...style,
+            cX, cY
+        }
+    })
+    
+    // 鼠标移动
+    const onMouseMove = useCallback((e) => {
+        // 判断鼠标是否按住
+        if (!isDown.current) return
+        let newStyle = transform(direction, oriPos, e);
+        setStyle(newStyle);
+    })
+
+    const onMouseUp = useCallback((e) => {
+      console.log(e, 'onMouseUp');
+      isDown.current = false;
+    }, [])
+
+    const constStyle = {
+      zIndex:'11', 
+      position:'fixed',
+    }
+
     const apiInstance = axios.create({
       baseURL: 'https://ct-dev.ncsa.illinois.edu',
```

Comment:
No hard-coded server names

Reply:
FYI in the latest version (as far as  i can tell using the github web interface, the hardcoded server name -  baseURL: 'https://ct-dev.ncsa.illinois.edu', is still in the latest version